src/context/fetch.ts

```diff
@@ -26,6 +26,21 @@ export const augmentRequestInit = (requestInit: RequestInit): RequestInit => {
   }
 }
 
+const createFetchRequestParameters = (input: MockedRequest) => {
+  const { body } = input
+  const requestParameters: RequestInit = {
+    ...input,
+    body: null,
+  }
+
+  if (input.method !== 'GET' && input.method !== 'HEAD') {
```

Comment:
Reversed logical operators are generally much harder to read (`!a` than `a`), so I recommend to flip this condition like so:

```js
// If it's a GET/HEAD request, return the request parameters as-is.
if (['GET', 'HEAD'].includes(method)) {
  return requestParameters
}

// Otherwise handle the request body.
requestParameters.body = typeof body === 'object' ? JSON.stringify(body) : body

return requestParameters
```